<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qSlicerVolumeReconstructionModuleWidget</class>
 <widget class="qSlicerWidget" name="qSlicerVolumeReconstructionModuleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="ctkCollapsibleButton" name="CTKCollapsibleButton">
     <property name="text">
      <string>Display</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QGroupBox" name="gridGroupBox">
        <property name="title">
         <string>Input</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Rotations File:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QPushButton" name="loadRotationsButton">
           <property name="text">
            <string>Load Rotations</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="rotationsLineEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Translations File:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Images:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Calibration Transformation:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QPushButton" name="loadTranslationsButton">
           <property name="text">
            <string>Load Translations</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="translationsLineEdit"/>
         </item>
         <item row="5" column="1">
          <widget class="qMRMLNodeComboBox" name="imagesMRMLNodeComboBox"/>
         </item>
         <item row="6" column="1">
          <widget class="qMRMLNodeComboBox" name="transformationMRMLNodeComboBox"/>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Volume Scale:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QSpinBox" name="volumeScaleSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10</number>
           </property>
           <property name="value">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item row="9" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Window Size:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <widget class="QSpinBox" name="windowSizeSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>50</number>
           </property>
           <property name="value">
            <number>3</number>
           </property>
          </widget>
         </item>
         <item row="10" column="1">
          <widget class="QCheckBox" name="holeFillingCheckBox">
           <property name="text">
            <string>Hole Filling</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Scale Transformation:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="qMRMLNodeComboBox" name="scaleMRMLNodeComboBox"/>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="5" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gridGroupBox">
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="spacing">
       <number>-1</number>
      </property>
      <item row="0" column="0">
       <widget class="QPushButton" name="reconstructionButton">
        <property name="text">
         <string>Generate 3D Reconstruction</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
    <slot>setMRMLScene(vtkMRMLScene*)</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>qSlicerWidget</class>
   <extends>QWidget</extends>
   <header>qSlicerWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkCollapsibleButton</class>
   <extends>QWidget</extends>
   <header>ctkCollapsibleButton.h</header>
   <container>1</container>
   <slots>
    <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>qSlicerVolumeReconstructionModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>imagesMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>234</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerVolumeReconstructionModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>transformationMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>485</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reconstructionButton</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerVolumeReconstructionModuleWidget</receiver>
   <slot>generateReconstruction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>253</x>
     <y>455</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadRotationsButton</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerVolumeReconstructionModuleWidget</receiver>
   <slot>loadRotationsFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>489</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>loadTranslationsButton</sender>
   <signal>clicked()</signal>
   <receiver>qSlicerVolumeReconstructionModuleWidget</receiver>
   <slot>loadTranslationsFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>194</y>
    </hint>
    <hint type="destinationlabel">
     <x>77</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>qSlicerVolumeReconstructionModuleWidget</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>scaleMRMLNodeComboBox</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>281</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>379</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loadRotationsFile()</slot>
  <slot>loadTranslationsFile()</slot>
  <slot>generateReconstruction()</slot>
 </slots>
</ui>
